version: '3.8'

services:
  web:
    build: .
    ports:
      - "7860:7860"
    volumes:
      - ./data:/data
    environment:
      - DATABASE_URL=postgresql://postgres.xqfmkhrlogcjpmxdnqky:lOw8qrlmpdpSYwVKSiWYhy_flHU0DbzUBbiUAa48Ul8@aws-0-us-west-1.pooler.supabase.com:5432/postgres
      - SUPABASE_URL=https://xqfmkhrlogcjpmxdnqky.supabase.co
      - SUPABASE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InhxZm1raHJsb2djanBteGRucWt5Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3NDk0MTYzODUsImV4cCI6MjA2NDk5MjM4NX0.lOw8qrlmpdpSYwVKSiWYhy_flHU0DbzUBbiUAa48Ul8
    command: ["python", "app.py"]

volumes:
  postgres_data: 